static/js/containers/Payment_test.js

```diff
@@ -1,90 +1,186 @@
+/* global SETTINGS: false */
 import React from 'react';
 import configureTestStore from 'redux-asserts';
 import { mount } from 'enzyme';
 import { Provider } from 'react-redux';
 import { assert } from 'chai';
 import sinon from 'sinon';
+import _ from 'lodash';
 
 import * as api from '../lib/api';
-import { setTotal } from '../actions';
+import {
+  setPaymentAmount,
+  setSelectedKlassIndex
+} from '../actions';
 import rootReducer from '../reducers';
 import Payment from '../containers/Payment';
 import * as util from '../util/util';
-import { makeRequestActionType, makeReceiveSuccessActionType } from '../rest';
+import {
+  makeRequestActionType,
+  makeReceiveSuccessActionType
+} from '../rest';
 
 const REQUEST_PAYMENT = makeRequestActionType('payment');
 const RECEIVE_PAYMENT_SUCCESS = makeReceiveSuccessActionType('payment');
+const REQUEST_KLASSES = makeRequestActionType('klasses');
+const RECEIVE_KLASSES_SUCCESS = makeReceiveSuccessActionType('klasses');
```

Comment:
something else I did in MM (with the redux-rest stuff) is stick the generated action types on the generated API action dispatcher functions as properties, so that you can do something like `actions.automaticEmails.get.successType`. We may want to do something similar over in bootcamp, but we don't have to right now I think.